web/src/engine/websites/LezhinES.ts

```diff
@@ -0,0 +1,11 @@
+import { Tags } from '../Tags';
+import icon from './LezhinES.webp';
+import Delitoon from './Delitoon';
+export default class extends Delitoon {
```

Comment:
LezhinES is based on Delitoon and not affiliated with LezhinEN, LezhinKO and LezhinJA?

Reply:
Exactly. Weird isnt it?